api/test/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hydrofoil/hypertest
+
+ADD / tests/
```

Comment:
The e2e test image is simply the base with test sources inserted

Reply:
Couldn't we mount this instead of building an image?

```diff
  e2e-tests:
-   build: api/test
+   image: hydrofoil/hypertest
    network_mode: "host"
+   volumes:
+     - api:/tests
```

Reply:
Sounds like an obvious solution if I honestly knew what I was doing 🙃